app/src/main/java/com/lightricks/feedexercise/ui/feed/FeedFragment.kt

```diff
@@ -37,14 +36,14 @@ class FeedFragment : Fragment() {
     }
 
     private fun setupViewModel() {
-        viewModel = ViewModelProvider(this, FeedViewModelFactory())
+        viewModel = ViewModelProvider(this, FeedViewModelFactory(activity!!))
             .get(FeedViewModel::class.java)
 
-        viewModel.getFeedItems().observe(viewLifecycleOwner, Observer { items ->
+        viewModel.getFeedItems().observe(viewLifecycleOwner, { items ->
```

Comment:
When I do it I'm getting a Type mismatch. Is it fixed in a newer version of Kotlin?